### PR TITLE
Clean ups following #1756

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: 107/src/main/java/org/ehcache/jsr107/Eh107InternalCacheManager.java [hunk 1]
```diff
@@ -30,11 +30,11 @@
  */
 class Eh107InternalCacheManager extends EhcacheManager {
 
-  public Eh107InternalCacheManager(Configuration config, Collection<Service> services, boolean useLoaderInAtomics) {
+  Eh107InternalCacheManager(Configuration config, Collection<Service> services, boolean useLoaderInAtomics) {
     super(config, services, useLoaderInAtomics);
   }
 
-  public ServiceLocator getServiceLocator() {
+  ServiceProvider<Service> getServiceLocator() {
```
Comment (henri-tremblay) [inline 107/src/main/java/org/ehcache/jsr107/Eh107InternalCacheManager.java]: Maybe also rename the method so it fits the return type?

File: 107/src/main/java/org/ehcache/jsr107/Eh107MXBean.java [hunk 2]
```diff
@@ -28,17 +28,16 @@
...
 
   private String sanitize(String string) {
     return string == null ? "" : string.replaceAll(",|:|=|\n", ".");
   }
 
-  private ObjectName createObjectName(String cacheName, Eh107CacheManager cacheManager, String beanName) {
-    URI uri = cacheManager.getURI();
-    String cacheManagerName = sanitize(uri != null ? uri.toString() : "null");
+  private ObjectName createObjectName(String cacheName, URI cacheManagerURI, String beanName) {
+    String cacheManagerName = sanitize(cacheManagerURI != null ? cacheManagerURI.toString() : "null");
```
Comment (henri-tremblay) [inline 107/src/main/java/org/ehcache/jsr107/Eh107MXBean.java]: I just been wondering. Can the cache manager URI really be null? Or it should be a precondition that it isn't?

File: 107/src/main/java/org/ehcache/jsr107/Eh107InternalCacheManager.java (hunk 1 above)
Comment (ljacomet) [inline 107/src/main/java/org/ehcache/jsr107/Eh107InternalCacheManager.java]: Fixing

File: 107/src/main/java/org/ehcache/jsr107/Eh107MXBean.java (hunk 2 above)
Comment (ljacomet) [inline 107/src/main/java/org/ehcache/jsr107/Eh107MXBean.java]: Indeed, `URI` is used as key in a map, can't be null. Cleaning up code

**Pull request merged**